pkg/network/setup/network.go

```diff
@@ -97,3 +120,16 @@ func (n *VMNetworkConfigurator) SetupPodNetworkPhase2(domain *api.Domain) error
 	}
 	return nil
 }
+
+func isIfaceAlreadyAvailableInVM(ifacesToHotplug map[string]struct{}, networkName string) bool {
+	_, found := ifacesToHotplug[networkName]
+	return len(ifacesToHotplug) > 0 && !found
+}
+
+func indexedInterfacesToHotplug(ifaceNames []string) map[string]struct{} {
```

Comment:
The method has nothing to do with hot plug `indexedInterfaces` should be enough.

Reply:
Done.